src/Command/Playercount.js

```diff
@@ -0,0 +1,114 @@
+    'errorBadKey': 'Hmm.. I couldn\'t find the game you were talking about.  Try again?',
+    'result': 'There are currently %1 players connected to %2.',
+    'help': 'You can use `!playercount` to show the players connected to the network or some of the games.\nYou can use any of the following names: %1'
+};
+
+const keyAlias = {
+    'hcf': 'hcfactions',
+    'anni': 'annihilation',
+    'civ-war': 'civwar',
+    'civcraft': 'civwar',
+    'minetheftauto': 'mta',
+    'gc': 'ghostcraft',
+    'ghost': 'ghostcraft',
+    'mine theft auto': 'mta',
+};
+
+// Map of key to name - also acts as a whitelist
+const gameNames = {
+    'all': 'Shotbow',
+    'annihilation': 'Annihilation',
+    'minez': 'MineZ',
+    'hcfactions': 'HCFactions',
+    'dbv': 'DBV',
+    'smash': 'SMASH',
+    'slaughter': 'Slaughter',
+    'civwar': 'Civ-War',
+    'gg': 'gg',
+    'mta': 'Mine Theft Auto',
+    'ghostcraft': 'GhostCraft',
+    'lobby': 'Lobby',
+};
+
+const cacheTTL = 10 * 1000; // 10 seconds in milliseconds
+
+module.exports = Command.extend({
+    commandName: 'playercount',
+    https: null,
+    cache: null,
+    cacheKey: 'serverlist',
+    dependencies: {
+        'discordClient': 'discordClient',
+        'commandPrefix': 'commandPrefix',
+        'https': 'https',
+        'Cache': 'cache'
+    },
+    processMessage: function (message, tokens) {
+        this.fetchServerlist(serverList => {
+            if (serverList === false) {
+                message.channel.send(messages.errorChecking);
+                return;
+            }
+            tokens.shift();
+            let key = this.getGameKey(tokens.join(' '));
+
+            if (key === 'help') {
+                let list = Object.values(gameNames)
+                    .map(item => { return '`' + item + '`'})
+                    .join(', ');
+                message.channel.send(messages.help.replace('%1', list));
+                return;
+            }
+
+            if (typeof gameNames[key] === 'undefined' || typeof serverList[key] === 'undefined') {
+                message.channel.send(messages.errorBadKey);
+                return;
+            }
+
+            let gameName = gameNames[key];
+            let count = serverList[key];
+
+            message.channel.send(messages.result.replace('%1', count).replace('%2', gameName));
+        });
+    },
+    getGameKey: function (requestedGame) {
+        if (!requestedGame) {
+            return 'all';
+        }
+        requestedGame = requestedGame.toLowerCase();
+        if (keyAlias[requestedGame]) {
+            requestedGame = keyAlias[requestedGame];
+        }
+        return requestedGame;
+    },
+    fetchServerlist: function (callback) {
+        let cachedData = this.cache.get(this.cacheKey);
+        if (cachedData) {
+            callback(cachedData);
+            return;
+        }
+        this.https.get('https://shotbow.net/serverList.json', res => {
```

Comment:
@JamsheedMistri  Is there a case here for if the connection fails I should be listening for?

Reply:
Yes. If `https://shotbow.net/serverList.json` is down for some reason, `https` will throw an error. Please catch this error and send a message to the player if caught.